code/modules/reagents/oldchem/reagents/drink/reagents_drink.dm

```diff
@@ -148,6 +148,24 @@
 	if(M.getBruteLoss() && prob(20)) M.heal_organ_damage(1,0)
 	if(holder.has_reagent("capsaicin"))
 		holder.remove_reagent("capsaicin", 2)
+
+	// Sometimes Crazylemon feels very silly
+	if(istype(M, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = M
+		if(H.species.name in list("Skeleton"))
```

Comment:
standard way of doing this in the code would generally be if(H.get_species() == "Skeleton")